core/controller/src/main/scala/whisk/core/controller/Activations.scala

```diff
@@ -159,40 +157,16 @@ trait WhiskActivationsApi extends Directives with AuthenticatedRouteProvider wit
       'upto.as[Instant] ?) { (skip, limit, count, docs, name, since, upto) =>
       if (count && !docs) {
         countEntities {
-          WhiskActivation.countCollectionInNamespace(
-            activationStore,
-            name.flatten.map(p => namespace.addPath(p)).getOrElse(namespace),
-            skip.n,
-            since,
-            upto,
-            StaleParameter.UpdateAfter,
-            viewName = name.flatten.map(_ => WhiskActivation.filtersView).getOrElse(WhiskActivation.view))
+          activationStore.countActivationsInNamespace(name.flatten, namespace, skip.n, since, upto)
```

Comment:
For some reason `name` is defined as a `Option[Option[EntityName]]`...

Reply:
That's because of

`'name.as[Option[EntityPath]] ?` which makes the Option optionable. @rabbah any reason why this is the case?